README.md

```diff
@@ -46,6 +49,96 @@ In your template you can do
 {{user.twitter}}
 ```
 and it will populate once the full user is loaded
+
+## How to use
+If you haven't customized any of your adapters, serializers or store, the addon will work out of the box. By default is uses JSONAPISerializer and JSONAPIAdapter.
+
+If you made any customization, you will need to include proper mixins. For adapter:
+
+```js
+// app/adapters/application.js
+
+import DS from 'ember-data';
+import PartialModelAdapter from 'ember-data-partial-model/mixins/adapter';
+const { RESTAdapter } = DS;
```

Comment:
I would default this example to the `JSONAPIAdapter` as default in Ember Data now.